lib/vagrant-service-manager/services/openshift.rb

```diff
@@ -0,0 +1,37 @@
+module Vagrant
+  module ServiceManager
+    module Service
+      class OpenShift
+        def initialize(app, env)
+          @app = app
+          @machine = env[:machine]
+          @ui = env[:ui]
+          @services = @machine.config.servicemanager.services.split(',').map(&:chomp)
+        end
+
+        def call(env)
+          if SUPPORTED_BOXES.include? @machine.guest.capability(:flavor)
+            if @machine.guest.capability(:flavor) == "cdk"
+              # openshift to be started by default for CDK
+              self.start_enable_openshift
+            elsif @services.include? "openshift"
+              # Its ADB and start openshift service if it is configured in Vagrantfile
+              self.start_enable_openshift
+            end
+          end
+          @app.call(env)
+        end
+
+        def start_enable_openshift
+          command = "sudo systemctl enable openshift && sudo systemctl restart openshift"
```

Comment:
The command should specifically either setup an environment for the shell to be used or call the command with explicit variables.  No variable should be passed unless it is provided by the user.  See the mods to README.md for notes.

`sudo DOCKER_REGISTRY=#{DOCKER_REGISTRY} IMAGE_TAG=#{IMAGE_TAG} IMAGE_NAME=#{IMAGE_NAME} /usr/bin/sccli openshift`

The variables are:
* DOCKER_REGISTRY
* IMAGE_NAME
* IMAGE_TAG